lcservice-go/service/core.go

```diff
@@ -145,22 +145,15 @@ func (r *commandHandlerResolver) preHandlerHook(request Request) error {
 	if err != nil {
 		return err
 	}
-	cid, err := request.GetCommandID()
-	if err != nil {
-		return err
-	}
 
 	// Test compat, ignore if no SDK.
 	if request.Org == nil {
 		return nil
 	}
 
 	if _, err := request.Org.Comms().Room(rid).Post(lc.NewMessage{
-		Type: lc.CommsMessageTypes.CommandAck,
-		Content: Dict{
-			"cid":     cid,
-			"command": request.Event.Data,
-		},
+		Type:    lc.CommsMessageTypes.CommandAck,
+		Content: request.Event.Data,
```

Comment:
There might be sensitive data in there? Can you make sure that nothing is leaked that shouldn't

Reply:
Saw this right after merging; the contents in description don't look sensitive to me, but I could be wrong? Is `ssid` private?